packages/wallets/src/wallets/types.ts

```diff
@@ -101,6 +101,7 @@ export type WalletArgsFor<C extends Chain> = {
     owner?: string;
     plugins?: WalletPlugin<C>[];
     options?: WalletOptions;
+    delegatedSigners?: C extends "solana" ? Array<DelegatedSigner> : never;
```

Comment:
Why limit to Solana though? We support this for all chains

Reply:
> These are minimal changes to release for a client commitment, further work in this area includes:
> Adding this for different chains (EVM, Stellar)

Making scope here as tight as possible to ship asap. Otherwise I'd need to spend the time testing all the other chains, and dealing with EVM's seemingly distinct signer format when specifying delegated signers on creation